packages/core/src/codewhisperer/service/securityScanHandler.ts

```diff
@@ -180,16 +186,16 @@ export async function getPresignedUrlAndUpload(
             },
         },
     }
-    getLogger().verbose(`Prepare for uploading src context...`)
+    logger.verbose(`Prepare for uploading src context...`)
     const srcResp = await client.createUploadUrl(srcReq).catch(err => {
         getLogger().error(`Failed getting presigned url for uploading src context. Request id: ${err.requestId}`)
         throw err
     })
-    getLogger().verbose(`Request id: ${srcResp.$response.requestId}`)
-    getLogger().verbose(`Complete Getting presigned Url for uploading src context.`)
-    getLogger().verbose(`Uploading src context...`)
-    await uploadArtifactToS3(zipMetadata.zipFilePath, srcResp)
-    getLogger().verbose(`Complete uploading src context.`)
+    logger.verbose(`Request id: ${srcResp.$response.requestId}`)
+    logger.verbose(`Complete Getting presigned Url for uploading src context.`)
+    logger.verbose(`Uploading src context...`)
```

Comment:
best practice is to build 1 string and log it, instead of logging multiple consecutive strings.